<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="filterBox">
            <property name="placeholderText">
             <string>Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearFilterButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ProjectView" name="treeView">
          <property name="showDropIndicator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::IgnoreAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="ProjectEditWidget" name="editTabs">
       <widget class="QWidget" name="tabWidgetPage1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <attribute name="title">
         <string/>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuClients">
    <property name="title">
     <string>Clients</string>
    </property>
    <addaction name="actionClientEdit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionReports"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuClients"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionAddSubproject"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionArchive"/>
   <addaction name="actionShow_Archived"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClientEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new</normaloff>:/new</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/delete</normaloff>:/delete</iconset>
   </property>
   <property name="text">
    <string>Delete Project</string>
   </property>
  </action>
  <action name="actionAddSubproject">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/subproject</normaloff>:/subproject</iconset>
   </property>
   <property name="text">
    <string>Add Subproject</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/start</normaloff>:/start</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/stop</normaloff>:/stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionShow_Archived">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/show_archived</normaloff>:/show_archived</iconset>
   </property>
   <property name="text">
    <string>Show Archived</string>
   </property>
  </action>
  <action name="actionArchive">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/archived</normaloff>:/archived</iconset>
   </property>
   <property name="text">
    <string>(Un)archive Project</string>
   </property>
   <property name="toolTip">
    <string>Archive or Unarchive Selected Project</string>
   </property>
  </action>
  <action name="actionReports">
   <property name="text">
    <string>Reports</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProjectView</class>
   <extends>QTreeView</extends>
   <header>projectview.h</header>
  </customwidget>
  <customwidget>
   <class>ProjectEditWidget</class>
   <extends>QTabWidget</extends>
   <header>projecteditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
